### PR TITLE
fix: rename Safe Apps settings route

### PR DESCRIPTION
**Pull request by iamacook**
## What it solves

Adjusting the Safe Apps settings route

## How this PR fixes it

The Safe Apps settings router has been renamed to `/settings/safe-apps` and the title adjusted to match our lowercase styling.

## How to test it

Open the Safe Apps permissions settings and observe the title be "Safe Apps permissions" and route `/settings/safe-apps`

## Screenshots

image

### CODE REVIEW

File: src/config/routes.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const AppRoutes = {
     modules: '/settings/modules',
     index: '/settings',
     appearance: '/settings/appearance',
-    safeAppsPermissions: '/settings/safe-apps-permissions',
```
Comment (katspaugh) [inline src/config/routes.ts:20]: Did you change this by hand? Because it's supposed to generate a nested object here.

File: src/config/routes.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export const AppRoutes = {
...
+    safeApps: '/settings/safe-apps',
```
Comment (yagopv) [inline src/config/routes.ts:20]: Not sure why `settings/safe-apps/` is better than `settings/safe-apps-permissions`, perhaps `settings/safe-apps/permissions` if you don't like the current route?

Comment (yagopv) [reply src/config/routes.ts:20]: Or you think about if we add more settings in the future we can create subsections there?

**Review by yagopv — APPROVED** [review]

File: src/config/routes.ts (hunk 2 above)
Comment (iamacook) [reply src/config/routes.ts:20]: Exactly that. For the time being, I think `/settings/safe-apps` is sufficient (until we add more).

File: src/config/routes.ts (hunk 1 above)
Comment (katspaugh) [reply src/config/routes.ts:20]: Could you please mod the script to camelCase dash-separated paths?

**Review by katspaugh — APPROVED** [review]

**Comment by iamacook** [comment]
I'll merge this as it's a route change, confirmed working.

**Comment by liliya-soroka** [comment]
noted

**Pull request merged**